geopandas/tests/test_plotting.py

```diff
@@ -552,6 +566,37 @@ def test_plot(self):
         self.df.plot()
 
 
+class TestGeometryCollectionPlotting:
+    def setup_method(self):
+        coll1 = GeometryCollection(
+            [
+                Polygon([(1, 0), (2, 0), (2, 1)]),
+                LineString([(0.5, 0.5), (1, 1), (1, 0.5), (1.5, 1)]),
```

Comment:
Add a MultiLinestring instead? (to have this combo of a multi object in a collection)